packages/office-ui-fabric-react/src/components/KeytipLayer/KeytipLayer.base.tsx

```diff
@@ -589,11 +584,7 @@ export class KeytipLayerBase extends BaseComponent<IKeytipLayerProps, IKeytipLay
    */
   private _isCurrentKeytipAnAlias(keytipProps: IKeytipProps): boolean {
     const currKtp = this._keytipTree.currentKeytip;
-    if (
-      currKtp &&
-      (currKtp.overflowSetSequence || currKtp.persisted) &&
-      arraysEqual(keytipProps.keySequences, currKtp.keySequences)
-    ) {
+    if (currKtp && (currKtp.overflowSetSequence || currKtp.persisted) && arraysEqual(keytipProps.keySequences, currKtp.keySequences)) {
       return true;
     }
     return false;
```

Comment:
couldn't this just be a `return <the current if check>;`

Reply:
Yes...this was just auto-formatted but will change

Reply:
Ah typescript complains about some types not being right which is probably why it was like this in the first place. I'll try to fix it

Reply:
Not really worth fixing as now I have to put an === true in one of the conditions because it's an optional boolean. I think I will keep as is